build.gradle

```diff
@@ -346,16 +346,16 @@ dependencies {
   testCompile  group: 'org.springframework.statemachine', name: 'spring-statemachine-test', version: versions.springStatemachine
   testCompile 'com.github.hmcts:fortify-client:1.2.2'
   //pact contract testing
-  //contractTestCompile group: 'au.com.dius', name: 'pact-jvm-consumer-junit5_2.12', version: versions.pact
```

Comment:
contractTestCompile is not supported any more... the recommended usage is contractTestImplementation